clients/database/migrations/20230306182203_add_request_table.up.sql

```diff
@@ -0,0 +1,8 @@
+-- Add table for per proxyied request metrics
+CREATE TABLE IF NOT EXISTS proxied_request_metrics (
```

Comment:
Let's adjust the schema to:
```sql
-- Add table for per proxyied request metrics
CREATE TABLE IF NOT EXISTS proxied_request_metrics (
  id bigserial, -- analytic tables usually hit the 4 byte limit
  method_name character varying not null, -- same as text but often used to signify short strings to people and code generators reading the schema
  block_number bigint, 
  response_latency_milliseconds real, -- use floats since we don't need exact percision and indexes, math, etc is faster
  request_time timestamp without time zone NOT NULL -- it's best to never use timestampz (there are only a couple cases were I would)
);
```
Also, it's probably worth adding values for the json-rpc version, id, and jsonb column for the parameters the method is called with.

In addition, we should track IP address and maybe some other details about the request.

Reply:
Also, it may be good to add a batch id as well to track how often the rpc is serving batch requests -- though latency is duplicated in these cases

Reply:
+1 for the schema changes. Tracking ip address requires infra changes to add X-Forwarded-For headers to our external load balancers, I created tickets for that work that also includes tracking the ip address as well. Overtime we will want to track more things and can add migrations to update the schema, for now trying to keep it pared down to what is being used / iteratively develop it in a way that is flexible for adding more features.